parachain/primitives/core/src/outbound.rs

```diff
@@ -16,34 +16,31 @@ pub type FeeAmount = u128;
 pub type GasAmount = u128;
 pub type GasPriceInWei = u128;
 
+/// OutboundFee which covers the cost of execution on both BridgeHub and Ethereum
+#[derive(Copy, Clone, Default, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub struct OutboundFee<Balance: BalanceT> {
+	/// Cover the cost on BridgeHub
+	pub base_fee: Balance,
+	/// Cover the cost on Ethereum
+	pub delivery_fee: Balance,
```

Comment:
The struct already has the word `Fees` in it.

```suggestion
	pub delivery: Balance,
```